src/pages/volunteer/events/index.tsx

```diff
@@ -0,0 +1,112 @@
+// Page for repairers to view their assigned events
+import { useState } from "react";
+import Image from "next/image";
+
+import Box from "@/components/EventBox/box";
+import LoadingSpinner from "@/components/UI/loading-spinner";
+import { useEvents } from "@/hooks/events";
+import { Event } from "@/types";
+const Home = () => {
+  const [sortKey, setSortKey] = useState<string>("startDate");
+  const [searchWord, setSearchWord] = useState<string>("");
+  const [sortMethod, setSortMethod] = useState<string>("asc");
+
+  const { data: eventData, isLoading: isEventsLoading } = useEvents(
+    sortKey,
+    sortMethod,
+    searchWord
+  );
+
+  // For future use
+  // const { userId } = useAuth();
+
+  function formatDate(dateString: string): string {
+    const actualDate = new Date(dateString);
+    const day = actualDate.getDate().toString().padStart(2, "0");
+    const month = (actualDate.getMonth() + 1).toString().padStart(2, "0");
+    const year = actualDate.getFullYear().toString();
+
+    return `${day}/${month}/${year}`;
+  }
+
+  function createBox(
+    id: string,
+    name: string,
+    startDate: Date,
+    endDate: Date,
+    description: string,
+    location: string
+  ) {
+    // To do: filter based off users assigned events
+    if (name != "Elegant Granite Chair") {
```

Comment:
Is there a reason why there is a hardcoded condition here? Don't have to worry about filtering based on events as that will be purely done in the backend, repairers will only be able to see their assigned events only when they query the `/event` endpoint.